OpenShift/common.sh

```diff
@@ -43,3 +43,31 @@ if [ ${#PULL_SECRET} = 0 ]; then
   echo "Get a valid pull secret (json string) from https://cloud.openshift.com/clusters/install#pull-secret"
   exit 1
 fi
+
+
+WORKING_DIR=${WORKING_DIR:-"/opt/install-scripts"}
+# Ironic vars
+export IRONIC_IMAGE=${IRONIC_IMAGE:-"quay.io/metal3-io/ironic:master"}
+export IRONIC_INSPECTOR_IMAGE=${IRONIC_INSPECTOR_IMAGE:-"quay.io/metal3-io/ironic-inspector:master"}
+export IPA_DOWNLOADER_IMAGE=${IPA_DOWNLOADER_IMAGE:-"quay.io/metal3-io/ironic-ipa-downloader:master"}
+export COREOS_DOWNLOADER_IMAGE=${COREOS_DOWNLOADER_IMAGE:-"quay.io/openshift-metal3/rhcos-downloader:master"}
+export IRONIC_DATA_DIR="$WORKING_DIR/ironic"
```

Comment:
As it would be only used for the images, I'd say we can use the $WORKING_DIR variable instead

Reply:
using CACHED_IMAGE_DIR based in SCRIPTDIR